modules/core/src/main/java/org/apache/ignite/marshaller/MarshallerUtils.java

```diff
@@ -172,12 +190,11 @@ private static ClassSet classWhiteList(ClassLoader clsLdr) throws IgniteCheckedE
      * @return Black list of classes.
      */
     private static ClassSet classBlackList(ClassLoader clsLdr) throws IgniteCheckedException {
-        ClassSet clsSet = null;
+        ClassSet clsSet = new ClassSet();
 
-        String blackListFileName = IgniteSystemProperties.getString(IgniteSystemProperties.IGNITE_MARSHALLER_BLACKLIST);
+        String blackListFileName = IgniteSystemProperties.getString(IGNITE_MARSHALLER_BLACKLIST, DEFAULT_BLACKLIST_CLS_NAMES_FILE);
```

Comment:
Perhaps we should append default blacklist, not replace it.

Reply:
Fixed.